acceptance/bundle/deployment/bind/schema/test.toml

```diff
@@ -1,3 +1,30 @@
-Local = false
+Local = true
```

Comment:
I think you can just omit this setting, we run all test locally by default

Reply:
I found that `Local = true` is used in more tests now and a logical follow-up change would be to remove it from all tests then? On a second thought I think it is useful to explicitly state that the author of the test intended it to be running locally. Let me know if you think otherwise. Resolving for now.